shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -0,0 +1,572 @@
+is that, in practice, each owner will pledge an equal amount of stake, but this
+is not enforced by the system.
+
+\clearpage
+\subsection{Calculating Rewards in External Currencies}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Value} & \textbf{Description} & \textbf{Calculated as} \\\hline
+$R_A$ &  Rewards value in \ada{} &\\\hline
+$D$ &  Dollar Price per \ada{} &\\\hline
+$R_D$ &  Rewards value in dollars & $R_A \times D$ \\\hline
+\hline
+\end{tabular}
+\end{center}
+\caption{Calculating Monetary Rewards}
+\label{fig:monetary}
+\end{figure}
+
+\noindent
+As usual, rewards in \ada{} can easily be converted to a dollar or other equivalent using
+the current exchange rate, $D$, as shown % in Figure~\ref{fig:monetary}
+above.  For example, if the rewards are \ADA{10,000} and the exchange rate is
+\$0.04 (1 \ada{} is worth 4 cents).  Realising this value would require an exchange,
+of course.
+
+\clearpage
+\subsection{Worked Example}
+
+
+This example is taken from the \IOHK{} internal calculator at:
+\url{https://docs.google.com/spreadsheets/d/1c-KmCNBIMZjHN7wdsrNx3ac1zT8xEqodK1OyZxC_QIk/edit#gid=1365573139}
+
+\subsubsection*{Key Parameters}
+\includegraphics[height=6in]{RC1.pdf}
+
+\noindent
+This diagram shows the key parameters and settings for the rewards system, including the
+rewards that are distributed from the reserves at each epoch.
+
+
+\subsubsection*{Stakepool Parameters}
+\includegraphics[width=\textwidth]{RC2.pdf}
+\noindent
+This diagram shows the Stakepool-specific parameters, including the total controlled \ada{} and the
+division by owner(s) and delegators.  Here, 98.71\% of the Stakepool \ada{} is owned
+by delegators.  The Stakepool receives all of its optimal rewards, since it controls
+exactly the required perecentage of stake (1.00\%, corresponding to the limit set from
+the $k$ parameter).
+
+\subsubsection*{Calculated Rewards}
+
+\hspace{-0.65in}\begin{minipage}{\textwidth}
+  \includegraphics[width=1.2\textwidth]{RC3.pdf}
+  \includegraphics[width=1.2\textwidth]{RC4.pdf}
+\end{minipage}
+
+\noindent
+This diagrams shows the corresponding rewards that accrue to the owner(s) and delegators,
+plus a calculation of the total income that is received by the owners.  We will assume that the average
+and actual gross rewards are the same, i.e. if the Stakepool controls 1\% of the total \ada{} in
+circulation, it will produce exactly 1\% of the blocks, and receive exactly 1\% of the rewards.
+In total, the
+owners and delegators to this Stakepool would receive a net reward that was equivalent to 8\% per year
+(the ``staking yield'').  If there were 4 owners for the Stakepool, each would receive
+25\% of the operator rewards, i.e. $\ADA{158,121.42} \div 4 ~~=~~ \ADA{39,530.36}$.
+Similarly, a delegator contributing 10\% of the delegated stake would receive 10\% of
+the delegator reward, i.e., $\ADA{12,096,288.83} \div 10 ~~=~~ \ADA{1,209,628.89}$.
```

Comment:
we have to be very careful with the rounding, in general we use round to zero (floor in the positive case) to avoid overpaying. Any remaining value goes to the treasury (I think) or some other pot. This is defined in the Shelley spec.